pkg/cmd/server/api/types.go

```diff
@@ -284,6 +284,9 @@ type GoogleIdentityProvider struct {
 	ClientID string
 	// ClientSecret is the oauth client secret
 	ClientSecret string
+
+	// HostedDomain is the optional Google App domain to restrict logins to
+	HostedDomain string
```

Comment:
Is the format well known enough to validate well or would we be risking cutting off valid cases we don't anticipate?